src/view-components/dashboard/dashboard.css

```diff
@@ -17,6 +17,21 @@
   grid-template-columns: 1fr;
 }
 
+:global(.omrs-breakpoint-lt-tablet) .container {
+  margin: 5px;
+  overflow-x: scroll;
```

Comment:
Should this be `auto`?

I ask because I think it will always show a scrollbar, right? Maybe I am mistaken here.

Reply:
sure am making the changes